Ff154 css typed OM cleanup (#44945)
mdn/content: files/en-us/web/api/cssmathproduct/cssmathproduct/index.md

{{APIRef("CSS Typed Object Model API")}}{{SeeCompatTable}}

> The **`CSSMathProduct()`** constructor creates a new {{domxref("CSSMathProduct")}} object which creates a new {{domxref('CSSMathProduct')}} object which multiplies the arguments passed into it.

Reviewer: looks like a copy paste duplication error in the original
```suggestion
The **`CSSMathProduct()`** constructor creates a {{domxref("CSSMathProduct")}} object that multiplies the arguments passed into it.
```
